config/code_schools.yml

```diff
@@ -438,6 +438,13 @@
   va_accepted: false
   full_time: true
   hardware_included: false
-  
-    
-  
+
+- name: Codeup
+  url: http://codeup.com/
+  address1: 600 Navarro St., #350
```

Comment:
Ah! Now it's obvious. I'll update and resubmit as soon as I get home. Just copied and pasted the address from their site -- my humblest and most profuse apologies. Also, I was just using "https://localhost:3000" per the documentation. I'll try it with the "/code_schools" later.